S21/Garrett-Reed/garrett_lowercaser.py

```diff
@@ -0,0 +1,126 @@
+# 3/6/2021 - Garrett Reed - Initial Creation - Alpha#
+# Alpha version will be console-only #
+
+# Function to drive process
+def wordCaseDriver():
+    wordProcess(langEntry(), wordEntry())
+
+
+def langEntry():
+    userLang = input("Please select a language (en, en-US, en-IE, en-Latn, tr, az, ga, ga-IE, el, zh-Hans, ja, th): ")
+    langList = ["en", "en-US", "en-IE", "en-Latn", "tr", "az", "ga", "ga-IE", "el", "zh-Hans", "ja", "th"]
+    try:
+        match = langList.index(userLang)
+    except:
+        print("Error - Unknown Language")
+        quit()
+    return match
+
+
+# Accepts language codes directly instead of through the console

```

Comment:
For Python comments to one function or method, following maybe better:
def function():
    """comments for function
     param1: type+description
    """